node/Dockerfile.alpine.slim.tpl

```diff
@@ -2,17 +2,33 @@
 FROM #{FROM}
 
 ENV NODE_VERSION #{NODE_VERSION}
+ENV YARN_VERSION #{YARN_VERSION}
 
 # Install dependencies
 RUN apk add --no-cache libgcc libstdc++ libuv libcrypto1.0 libssl1.0
 
 RUN buildDeps='curl' \
 	&& set -x \
+	&& for key in \
+	6A010C5166006599AA17F08146C2130DFD2497F5 \
+	; do \
+		gpg --keyserver pgp.mit.edu --recv-keys "$key" || \
```

Comment:
Why do you fetch the key from multiple key servers?

Reply:
@petrosagg, I get this from upstream dockerfile, I think they do that since a key server might not be available all the time. It happens a lot in our builds when a build fails due to a key server was unreachable